plugins/modules/filesystem.py

```diff
@@ -40,7 +40,7 @@
     default: present
     version_added: 1.3.0
   fstype:
-    choices: [ btrfs, ext2, ext3, ext4, ext4dev, f2fs, lvm, ocfs2, reiserfs, xfs, vfat, swap, ufs ]
+    choices: [ bcachefs, btrfs, ext2, ext3, ext4, ext4dev, f2fs, lvm, ocfs2, reiserfs, xfs, vfat, swap, ufs ]
```

Comment:
Can you add another entry to the `description` below: `bcachefs support has been added in community.general 8.5.0.`?